site/pages/privacy-policy/policy.js

```diff
@@ -269,7 +257,7 @@ const policies = [
       {
         body: () => (
           <P>
-            <b>Right to withdraw consent:</b> To the extent that the legal basis for our processing
+            <B>Right to withdraw consent:</B> To the extent that the legal basis for our processing
```

Comment:
in line 288 of this file there is a word missing
`personal and` -> `personal data and`
could fix this while updating the file